source/common/formatter/substitution_format_string.cc

```diff
@@ -25,5 +26,18 @@ FormatterPtr SubstitutionFormatStringUtils::fromProtoConfig(
   return nullptr;
 }
 
+absl::string_view SubstitutionFormatStringUtils::getContentType(
+    const envoy::config::core::v3::SubstitutionFormatString& config) {
+  const std::string contentType = config.content_type();
```

Comment:
Simplify this as follows as those are the only ones supported now ? 

```
if (contentType == Http::Headers::get().ContentTypeValues.Html)
    return Http::Headers::get().ContentTypeValues.Html;
  else
    return Http::Headers::get().ContentTypeValues.Text;
```

Reply:
Why can't we plumb through the content type directly, without having to rely on well-known values?